src/ansi-c/expr2c.cpp

```diff
@@ -465,8 +485,10 @@ std::string expr2ct::convert_rec(
   else if(src.id()==ID_symbol)
   {
     symbol_typet symbolic_type=to_symbol_type(src);
-    const irep_idt &typedef_identifer=symbolic_type.get(ID_typedef);
+    if(!ns)
+      return q + id2string(symbolic_type.get_identifier());
 
+    const irep_idt &typedef_identifer = symbolic_type.get(ID_typedef);
```

Comment:
As this diff is pretty big, I'd appreciate if white-space changes could be refrained from. I know clang-format won't like it, but reviewing is a huge pain with those.

Reply:
Try appending `?w=1` on Github to omit whitespace-only changes, or plain old `git diff -w`

Reply:
Forcing other people to take a detour is not a terribly nice thing to do. This is not a show-stopper, but it would be nice if people were considerate of that when putting together pull requests.